src/dialogs/more-info/controls/more-info-light.ts

```diff
@@ -147,7 +147,7 @@ class MoreInfoLight extends LitElement {
           : ""}
         <ha-attributes
           .stateObj=${this.stateObj}
-          extra-filters="brightness,color_temp,white_value,effect_list,effect,hs_color,rgb_color,xy_color,min_mireds,max_mireds,entity_id"
+          .extraFilters=${"brightness,color_temp,white_value,effect_list,effect,hs_color,rgb_color,xy_color,min_mireds,max_mireds,entity_id"}
```

Comment:
Wow, this has been fixed 500 times now... But the old one should just work?

Reply:
And it does work here?

Reply:
I thought so too. But it doesnt in my dev

Reply:
It does here...

Reply:
It doesn't in production for sean

Reply:
or for me

Reply:
So you see all these attributes in the more info dialog?

Reply:
I do not

Reply:
Haha it is supposed to hide all these attributes, not to show them.

Reply:
Okay
Well
I clearly do not understand this element's purpose :)

Reply:
It is a filter of attributes not to show in ha attributes, check the code

Reply:
That makes sense I just never looked at it :)